pyccel/codegen/printing/cwrappercode.py

```diff
@@ -647,8 +900,8 @@ def _print_Module(self, expr):
                                      '}}').format(
                                             name = f.name,
                                             wrapper_name = self._function_wrapper_names[f.name],
-                                            doc_string = f.doc_string)
-                                     for f in expr.funcs)
+                                            doc_string = f.doc_string if isinstance(f, FunctionDef) else f.functions[0].doc_string)
```

Comment:
Would it not be simpler to add a `doc_string` property to `Interface`? It has getters for most of `FunctionDef`'s other properties